### ISSUE TITLE
Page names/links need to be renamed

### ISSUE THREAD

**Issue by monroemann** [monroemann/breakdivingnew#230]
Right now, all the page names are alphabetically poorly structured.  For example, all the google landing pages are spread out throughout the file tree.  It would be easier if they were all in a folder called, "Landing Pages", so that we don't confuse them with actual main site pages.  

But when we make this change, would we then need to change all the URLS?  

For example, I still want www.breakdiving.org/acting_chat to work.  I don't want to have to go into adwords and rename all the URLS: www.breakdiving.org/landing_pages/acting_chat...

Thoughts?

**Comment by chmb** [comment]
Yes each url will need to be renamed. Do you still want them in a separate folder?

**Comment by monroemann** [comment]
What is another way to do it?  Or is there no way to better organize them without renaming?
